src/Core/Content/Product/Helper/SalesChannelProductHelper.php

```diff
@@ -0,0 +1,100 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Product\Helper;
+
+use Shopware\Core\Content\Product\SalesChannel\SalesChannelProductEntity;
+use Shopware\Core\Content\Property\Aggregate\PropertyGroupOption\PropertyGroupOptionCollection;
+use Shopware\Core\Content\Property\PropertyGroupCollection;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Shopware\Core\System\SystemConfig\SystemConfigService;
+
+class SalesChannelProductHelper
```

Comment:
```suggestion
class SalesChannelProductBuilder
```

same as above move it out of the directory helper.